grid/social-media/replies/maps/linkedin.suma

```diff
@@ -26,15 +26,10 @@ map GetPostComments {
         return {
           id: element["$URN"],
           text: element.message.text,
-          created: {
-            by: element.created.actor,
-            at: element.created.time
-          },
-          lastModified: {
-            by: element.lastModified.actor,
-            at: element.lastModified.time
-          }
-        }
+          createdAt: undefined, // TODO once SDK 1.4 lands
+          authorId: element.created.actor,
+          lastModifiedAt: undefined // TODO once SDK 1.4 lands
```

Comment:
Please, fix the TODOs

Reply:
This was fixed in https://github.com/superfaceai/station/pull/259/commits/1d146a9b6c96f3ed5f5f05a9bf55ca623a06025a before you started the review, not sure why it didn't show up for you 🤔